src/triage/component/postmodeling/utils/config_file_example.yaml

```diff
@@ -0,0 +1,16 @@
+# The relevant experiment hash
```

Comment:
Might want to name this file something like `add_predictions_example_config.yaml` in case we end creating other utility functions with different config files.

Reply:
makes sense